### ISSUE TITLE
Error with -r main when pulling sarek

### ISSUE THREAD

**Issue by wigge206** [GenomicsAotearoa/Nextflow_Workshop#6]
The `-r main` tag when pulling sarek is causing an error, could specify the current release `-r 3.3.2`?

```
WARN: Cannot read project manifest -- Cause: Remote resource not found: https://api.github.com/repos/nf-core/sarek/contents/nextflow.config?ref=main
Remote resource not found: https://api.github.com/repos/nf-core/sarek/contents/main.nf?ref=main
```

**Comment by DininduSenanayake** [comment]
I believe this is because of `-main` as `nf-core/sarek` is still using the old branch nomenclature .i.e.` master` and not `main`. 

```bash
$ nextflow pull nf-core/sarek -r master
Checking nf-core/sarek ...
 downloaded from https://github.com/nf-core/sarek.git - revision: f034b73763 [master]
```

**Comment by DininduSenanayake** [comment]
fixed as per 7e911faa9cbf1ec1347085ee8eee18a5791b8d17
